Moq.nuspec

```diff
@@ -27,6 +27,15 @@
 				<dependency id="System.Threading.Tasks.Extensions" version="4.3.0" />
 				<dependency id="System.ValueTuple" version="4.4.0" />
 			</group>
+			<group targetFramework=".NETStandard2.0">
```

Comment:
If we do indeed target `netcoreapp2.x` instead the dependencies will likely look a bit more minimal. This will likely need to be changed. Stay tuned for more feedback later.